jerry-core/ecma/operations/ecma-conversion.cpp

```diff
@@ -102,22 +102,30 @@ ecma_op_same_value (ecma_value_t x, /**< ecma-value */
   {
     return false;
   }
-
-  if (is_x_undefined
-      || is_x_null)
+  else if (is_x_undefined
```

Comment:
Condition is too simple to move it on other line. The only reason to use multiline condition - it doesn't fit into 120.